### ISSUE TITLE
How to reduce the audio only rtsp latency?

### ISSUE THREAD

**Issue by MikeXtremeLED** [aler9/rtsp-simple-server#145]
## Which version are you using?

v0.12.2

## Which operating system are you using?

Windows 

## Question
How do reduce the rtsp latency on a audio stream? I got now 2-3 sec delay. I would like to have something like 0,5-1sec. 

Description

**Comment by aler9** [comment]
Hello, rtsp-simple-server by itself doesn't introduce any delay (maybe 1-2ms, not more), delay is introduced

* by the publisher (which software are you using to publish?)
* by the reader, for buffering purposes - for instance, VLC adds a 1 second buffer to compensate bandwidth fluctuations - to disable this feature, insert a low value (10ms) into the "Caching" field that can be seen in the "Open media" window

![Screenshot_20201213_000635](https://user-images.githubusercontent.com/46489434/101998869-5fedfe80-3cd7-11eb-8eb6-39fd161e513e.jpg)

**Comment by MikeXtremeLED** [comment]
Nicee thanks for the info. 


I use a Enocoder from the company Uray. 

Use it with them HDMI encoder to make a rtsp push. 

For decoding I use mobile phones. iPhone and android. The idea of the rtsp server is to hear the intercom audio feed from the studio. That the 4g cameraman knows what to do. 

![image](https://user-images.githubusercontent.com/75535496/102726125-4c1c4b00-431c-11eb-97ce-7c3a10493636.jpeg)

On the iPhone I found the low latency function. 

For android I also found a cache time. But the delay time keeps the same when I set it from 1000 to 10.
